Tests/ApodiniTests/PropertiesTests/BindingTests.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  BindingTests.swift
+//  
+//
+//  Created by Max Obermeier on 24.02.21.
+//
+
+import Foundation
+
+@testable import Apodini
+import XCTest
+import XCTApodini
+import ApodiniREST
+
+final class BindingTests: ApodiniTests, EnvironmentAccessible {
+    struct Greeter: Handler {
+        @Binding var country: String?
+        
+        func handle() -> String {
+            country ?? "World"
+        }
+    }
+    
+    struct ReallyOptionalGreeter: Handler {
+        @Binding var country: String??
+        
+        func handle() -> String? {
+            if let country = self.country {
+                return country ?? "World"
+            } else {
+                return nil
+            }
+        }
+    }
+        
+    var featured: String? = "Italy"
+    
+    @PathParameter var selectedCountry: String
+    @Environment(\BindingTests.featured) var featuredCountry
+    
+    @ComponentBuilder
+    var testService: some Component {
+        Greeter(country: .constant(nil))
+        Group("default") {
+            Greeter(country: Binding<String?>(.constant("USA")))
+        }
+        Group("optional") {
+            Greeter(country: Parameter<String?>().projectedValue)
```

Comment:
> But from looking at the idea of Bindings I think it might make sense to have a Component like this: ...

I agree. I changed the test to reflect this.